tachyon/crypto/sumcheck/multilinear/BUILD.bazel

```
@@ -0,0 +1,14 @@
+load("//bazel:tachyon_cc.bzl", "tachyon_cc_library")
+
+package(default_visibility = ["//visibility:public"])
+
+tachyon_cc_library(
+    name = "multilinear_sumcheck",
```

Are you going to add more stuffs here? Otherwise, please name it `data_structures`.

```suggestion
    name = "data_structures",
```

---

The actual multilinear sumcheck will be implemented like [so](https://github.com/arkworks-rs/sumcheck/blob/master/src/ml_sumcheck/mod.rs) later, but you're right. I'll change this to "data_structures" and create a different tachyon_cc_library label later for the actual multilinear_sumcheck.